flycheck.el

```diff
@@ -3942,8 +3967,10 @@ Return a list with the contents of the table cell."
                    column 'flycheck-error-list-column-number)
                   (flycheck-error-list-make-cell
                    (symbol-name (flycheck-error-level error)) level-face)
-                  (flycheck-error-list-make-cell
-                   id-str 'flycheck-error-list-id id-str)
+                  ;; Error ID has a button to explain the error
+                  (list id-str
+                        'type 'flycheck-error-list-explain-error
```

Comment:
@fmdkdd @cpitclaudel You lost me somewhere in the midst of your exchange :blush:

The TL;DR is that the error ID column is now a proper button if an explainer is present and a plain text otherwise?